server/views/pages/overview/partials/educationAndTrainingOtherQualifications.njk

```diff
@@ -1,34 +1,27 @@
-{% if not prePrisonQualifications.problemRetrievingData %}
+{% if not otherQualifications.problemRetrievingData %}
 
   <div class="govuk-grid-row">
     <div class="govuk-grid-column-two-thirds">
 
       <div class="govuk-summary-card">
         <div class="govuk-summary-card__title-wrapper">
-          <h2 class="govuk-summary-card__title">Pre-prison qualifications and education</h2>
+          <h2 class="govuk-summary-card__title">Other qualifications and history</h2>
         </div>
         <div class="govuk-summary-card__content">
-          <table class="govuk-table" id="pre-prison-qualifications-table">
-            <caption class="govuk-table__caption govuk-visually-hidden">Details pre-prison qualifications and education that have been completed</caption>
-
-            <thead class="govuk-table__head">
-              <tr class="govuk-table__row">
-                <th scope="col" class="govuk-table__header">Type</th>
-                <th scope="col" class="govuk-table__header">Completed on</th>
-              </tr>
-            </thead>
+          <table class="govuk-table" id="other-qualifications-table">
```

Comment:
I might be wrong, but I'm not sure this is a govuk table anymore, given that we've removed the header row?
I wonder if a [summary list](https://design-system.service.gov.uk/components/summary-list/) is more appropriate now?

@chrimesdev ; one for your expertise! - table or summary list?

Reply:
Amazing; thanks Adam 👍